data/json/regional_map_settings.json

```diff
@@ -58,7 +58,8 @@
         "f_dandelion": 8,
         "f_boulder_small": 1,
         "f_boulder_medium": 0.6666,
-        "f_boulder_large": 0.3333
+        "f_boulder_large": 0.3333,
+        "f_shrub_bee_hive": 1
```

Comment:
I may be reading this wrong, but bee hive shrub would probably have to be much rarer than that. Probably at most about 1 bee shrub per 50 normal shrubs (of all kinds), if you're looking for them deliberately rather than disturbing one at random while foraging.

Reply:
It's four times rarer than rose or hydrangea bushes, but ~2 times more frequent than blueberry or strawberry bushes. I'm open to suggestions on how to balance that.

Reply:
Well, I bet bee hives are significantly less common than berry bushes of any kind. That's why people don't really go to get honey in forest; most people collect mushrooms or berries instead.